endless/src/endless/EndlessUsbToolDlg.cpp

```diff
@@ -5043,7 +5053,8 @@ bool CEndlessUsbToolDlg::SetupComboDisk(HANDLE hPhysical)
 error:
     return result;
 }
-bool CEndlessUsbToolDlg::CreateUSBPartitionLayout(HANDLE hPhysical)
+
+bool CEndlessUsbToolDlg::CreateUSBPartitionLayout(HANDLE hPhysical, bool just_esp)
```

Comment:
It's a minor thing and not worth revving this patch series for but I don't like boolean parameters. Perhaps:

```c++
enum CreatePartitionMode {
    JUST_ESP,
    EVERYTHING
}
```